puma/hlplots/tagger.py

```diff
@@ -82,6 +88,35 @@ def variables(self):
         """
         return [f"{self.name}_{prob}" for prob in self.probabilities]
 
+    @property
+    def aux_variables(self):
+        """Return a dict of the auxiliary outputs of the tagger for each task.
+
+        Returns
+        -------
+        aux_outputs: dict
+            Dictionary of auxiliary output variables of the tagger
+
+        Raises
+        ------
+        ValueError
+            If element in self.aux_tasks is unrecognized
+        """
+        aux_outputs = {}
+
+        for aux_type in self.aux_tasks:
+            if aux_type == "vertexing":
+                if self.name == "SV1" or self.name == "JF":
```

Comment:
Hard-coded exceptions for SV1 and JetFitter due to inconsistency in variable naming. Might also make sense to move this upstream.

Reply:
Again, happy with this here for now unless we need the same logic in another package